app/src/renderer/components/staking/TabMyDelegations.vue

```diff
@@ -15,7 +15,7 @@
 
     div(v-if="undelegatedValidators.length")
       h3.tab-header
-        | Your Undelegated Validators
+        | Inactive Delegations
```

Comment:
```suggestion
        | Unbonding Delegations
```

Reply:
no B words @fedekunze ... let's call it `Undelegated Validators` for now?

Reply:
Well... technically this is the only b-word used on the SDK now. But `Undelegated Validators` sounds better 👍